tests/test_filters.py

```diff
@@ -739,6 +739,16 @@ def test_update_type_edited_channel_post(self, update):
         assert Filters.update.channel_posts(update)
         assert Filters.update(update)
 
+    def test_update_type_poll(self, update):
+        update.poll = True
```

Comment:
maybe `assert not Filters.update.poll(update)` first?

Reply:
Good idea

Reply:
Good idea